### PR TITLE
various documentation changes

### PR DESCRIPTION
**Pull request by z3dev**
Please see the list of commit messages for what has changed.

These changes do not contain any changes to functionality or fixes.

### All Submissions:

* [X] Have you followed the guidelines in our Contributing document?
* [X] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
* [X] Does your submission pass tests?

### CODE REVIEW

File: jsdoc/tutorials/02_modelingBasics.md [hunk 1]
```diff
@@ -38,9 +38,8 @@ Another valuable way of building shapes is to start with 2D shapes.  These can b
 specifying points, starting with basic primitive 2D shapes, or importing SVG files.  The 2D 
 shapes can then be extruded to produce a wide variety of different geometries.
 ```javascript
-const jscad = require('@jscad/modeling')
-const {polygon } = jscad.primitives
-const { extrudeLinear } = jscad.extrusions
+const { polygon } = require('@jscad/modeling').primitives
```
Comment (hrgdavor) [inline jsdoc/tutorials/02_modelingBasics.md:41]: not really a fan of calling require constantly, the old version with `const {polygon } = jscad.primitives` was more to my liking

**Review by hrgdavor — APPROVED** [review]
There is  comment regarding the style for imports, but it is a presonal preference, I'll leave it to you to decide.

Otherwise, changes look good

File: jsdoc/tutorials/02_modelingBasics.md (hunk 1 above)
Comment (z3dev) [reply jsdoc/tutorials/02_modelingBasics.md]: Done. We should also be consistent across documentation and examples as well.

**Review by mister-jscad — APPROVED** [review]
Looks fine.

**Pull request merged**